src/styles/GlobalStyle.tsx

```diff
@@ -0,0 +1,30 @@
+// src/styles/GlobalStyle.tsx
+import { Global, css } from '@emotion/react'
+
+const GlobalStyle = () => (
+  <Global
+    styles={css`
+      @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css');

```

Comment:
[심화]

이번 과제 필수 내용은 아니지만 다음 단계에 반영해보면 좋을 거 같습니다.

1. cdn 을 통한 리소스 요청 시 버전 입력
```suggestion
  @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.css');
```
 이렇게 하면 일단 브라우저에서 캐싱을 더 안정적으로 할 수 있고요, 보통 버전을 명시하지 않으면 최신 버전을 가져오는데 이렇게 되면 pretendard에서 변경 사항이 있는 경우 알아차리지 못한채로 서비스 동작이 달라질 수 있습니다. 그래서 cdn에서 데이터를 가져올 때는 버전을 명시해주는 것이 여러모로 안전해요!

2. css import 위치 이동

import 구문을 index.html 쪽으로 이동하시는 것을 권장드립니다! 

 이렇게 하면 js가 실행되기 전에 미리 cdn에게 CSS를 요청할 수 있어요. 하지만 여기서 import를 요청하게 되면 브라우저가 이 파일을 실행할 때까지 CSS 다운로드 요청이 일어나지 않기 때문에 로딩 성능에서 손해를 볼 수 있어요! FoUC (Flash of Unstyled Content) 문제가 발생할 확률도 높고요. 하지만 html head 안 쪽에 배치하면 js 파싱전에 미리 CSS 요청을 호출하므로 훨씬 효율적입니다! 

 사실 지금 단계에서는 성능에 큰 차이가 없을텐데요, FE 개발자 단골 면접 질문인 브라우저 렌더링 과정에 포함되는 내용이라서 한 번 짚어 드렸습니다! 수업 듣느라 바쁘시겠지만.. ㅠ 아래 포스트도 같이 읽어보시면 좋을 거 같아요.

https://developer.chrome.com/blog/inside-browser-part3?hl=ko